src/StdUtils.sol

```diff
@@ -74,6 +74,11 @@ abstract contract StdUtils {
         return addressFromLast20Bytes(keccak256(abi.encodePacked(bytes1(0xff), deployer, salt, initcodeHash)));
     }
 
+    function bytesToUint(bytes memory b) internal pure returns (uint256) {
+        require(b.length <= 32, "StdUtils bytesToUint(bytes): Bytes length exceeds 32.");
+        return abi.decode(abi.encodePacked(new bytes(32 - b.length), b), (uint256));
+    }
```

Comment:
Is there a way we can avoid duplicating the implementations, and have a single `bytesToUint` or have this one just call `_bytesToUint`? @ZeroEkkusu from what I gather it seems this was originally a library that was deduped but you prefer this version?

Reply:
If there's agreement,  I can put it back into a library and have both use the single implementation from there.

Reply:
`bytesToUint` in `StdUtils` is *the* implementation. The one in `StdCheats`, which has been renamed to `_bytesToUint`, is just a `private` helper function for some std-cheats. We do not care about that one, so to speak, because it's not an actual feature - it was used only during the development and we do not expect to ever revisit it. Does that make sense?

Reply:
Right, but they serve the same purpose so seems logical to dedupe. But tbh I don't feel strongly and I don't think deduping here is a big deal so happy to concede and not block on this